Update add-file-secrets.md (#7331)
harness/developer-hub: docs/platform/secrets/add-file-secrets.md

8. Select **Save**.

  :::warning

>    File secrets are only created as Inline secrets. Inline secrets if deleted from Harness will also delete the secrets from the external secrets manager. It is recommended to always take a backup of these secrets.

Reviewer: ```suggestion
   File secrets are only created as Inline secrets. Inline secrets deleted in Harness are also deleted from the external secrets manager. Harness recommends that you always back up these secrets.
```
